@ripoul you fix what hey report:
./superset/views/core.py:754:13: E122 continuation line missing indentation or outdented

that means that you have to readd the 4 spaces indentation you removed )

--- a/superset/views/core.py
+++ b/superset/views/core.py
@@ -751,7 +751,7 @@ class R(BaseSupersetView):
         scheme = request.scheme
         return Response(
             '{scheme}://{request.headers[Host]}/{directory}?r={obj.id}'.format(
-            scheme=scheme, request=request, directory=directory, obj=obj),
+                scheme=scheme, request=request, directory=directory, obj=obj),
             mimetype='text/plain')
 
     @expose('/msg/')

--- a/superset/views/core.py
+++ b/superset/views/core.py
@@ -748,10 +748,9 @@ class R(BaseSupersetView):
         obj = models.Url(url=url)
         db.session.add(obj)
         db.session.commit()
-        scheme = request.scheme
         return Response(
             '{scheme}://{request.headers[Host]}/{directory}?r={obj.id}'.format(
-                scheme=scheme, request=request, directory=directory, obj=obj),
+                scheme=request.scheme, request=request, directory=directory, obj=obj),
             mimetype='text/plain')
 
     @expose('/msg/')

